src/components/avatar_name/__snapshots__/index.test.tsx.snap

```diff
@@ -5,6 +5,7 @@ exports[`component: AvatarName matches snapshot 1`] = `
   href="/accounts/123"
   onClick={[Function]}
   onMouseEnter={[Function]}
+  onTouchStart={[Function]}
```

Comment:
Hey @MonikaCat I'm curious is this auto updated by yarn?

Reply:
Hey @yayay927 yes it was added by yarn after running yarn test cmd, can you test if it works as required? Thanks ^^

Reply:
Hey @MonikaCat after doing some research I found out it might be more possible to be added by jest rather then yarn. For the reason I didn't find much information related between the 2. Also, after viewing old codebase, I found out they existed before and this function is usually added into html link or button. Lastly, for the reason snapshot file is already a result of testing, it should already proves it works fine. 
Hope I didn't bring confusion for you ^^ I will merge this pr now